### DISCUSSION TITLE
申请数据库查询权限，没有表级别选项

### DISCUSSION THREAD

**Discussion by 17tiam** [hhyo/Archery#2501] [General]
版本：docker-compose 部署 v1.10.0版本
问题：如下图申请数据库查询权限，没有表级别选项，是需要配置什么吗
![87961706778968_ pic](https://github.com/hhyo/Archery/assets/17775511/3e5dc4f9-7374-41ca-b856-be1504eaf71e)

**Comment by LeoQuote** [comment]
重复 https://github.com/hhyo/Archery/issues/2289
